src/renderer/callbacks-registry.ts

```diff
@@ -11,12 +11,14 @@ export class CallbacksRegistry {
     if (id != null) return id
```

Comment:
perhaps explicitly specify the return type of `add` here so that the bug you're fixing here would be caught by the compiler?